server/routes/api/moderator/moderatorRoutes.ts

```diff
@@ -0,0 +1,11 @@
+import { Router } from "express";
+
+import Handler from "./moderatorHandler";
+
+const moderatorRouter = Router();
+
+const handler = new Handler();
+
+moderatorRouter.get("/", handler.getAllUsers);
```

Comment:
thinking out load - do we want to add specific path for all users?
currently if you address: `/api/moderator/` you'll get all users maybe let's use: `/api/moderator/allUsers/`

Reply:
good idea